rs/nervous_system/common/test_utils/src/wasm_helpers.rs

```diff
@@ -5,6 +5,11 @@ use std::io::Read;
 /// A small, valid WASM suitable for tests.
 pub const SMALLEST_VALID_WASM_BYTES: &[u8; 8] = &[0, 0x61, 0x73, 0x6D, 1, 0, 0, 0];
 
+pub const SMALLEST_VALID_WASM_HASH: [u8; 32] = [
```

Comment:
Nit

Maybe add a comment how this was computed. 

Alternatively, why not use lazy_static over a library call?